src/OpenToolkit.Mathematics/Geometry/Box2.cs

```diff
@@ -121,6 +121,160 @@ public Vector2 Center
             set => Translate(value - Center);
         }
 
+        // --
+
+        public float Width
```

Comment:
Should there be a way to change the Width and Height at the same time like with `Size` but only move the max instead of resizing from the center?